app/views/template_invocations/show.html.erb

```diff
@@ -28,7 +28,13 @@ end
   </div>
 </div>
 <% if @host %>
-  <h3><%= _('Target: ') %><%= link_to(@host.name, host_path(@host)) %></h3>
+  <% proxy_id = @template_invocation_task.input[:proxy_id] %>
```

Comment:
Where is the proxy_id comming from? Is that related to the `input[:proxy_id]`, or are we just exposing it on two independent places?